### PR TITLE
Ethereals now use alcohol as blood

### PR DESCRIPTION
**Pull request by wesoda25**
## About The Pull Request

Adds isopropyl alcohol to the game, a powerful, highly toxic (for non ethereals) reagent. Changes ethereal blood from liquid electricity to isopropyl alcohol. Makes alcohol have 0 effect on Ethereals.

## Why It's Good For The Game

I've never been a big fan of how Ethereals' blood is the same thing they eat to get charge. It's odd and leads to some bizarre interactions. I figured it would make sense that Ethereal blood would be a highly conductive liquid, so I looked up some of these, one of which is isopropyl alcohol.

This change is an interesting and unique aspect for Ethereals, and gives them immunity to alcohol (as most alien species probably would have). I think this is the sort of thing we need to make our various nonhuman species unique.

## Changelog
:cl:
add: Adds Isopropyl Alcohol, a very strong and highly toxic reagent for non-Ethereals.
tweak: Isopropyl Alcohol now acts as Ethereal blood, as opposed to liquid electricity. 
balance: Ethereals are now immune to alcohol's effects.
/:cl:

Floyd and mothblocks ok'd this

### CODE REVIEW

**Comment by wesoda25** [comment]
Floyd mentioned he'd like some interesting ways for Ethereals to regenerate blood. Lmk if you have any ideas for that.

**Comment by CRITAWAKETS** [comment]
> Floyd mentioned he'd like some interesting ways for Ethereals to regenerate blood. Lmk if you have any ideas for that.
make it so that drinking alcoholic drinks gives blood with booze power affecting the conversion ratio. So basically, an IV drip of bacchus' blessing will cause you to have 4000% blood.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -41,6 +41,8 @@ All effects don't start immediately, but rather get worse over time; the rate is
 			booze_power *= 0.7
 		if(HAS_TRAIT(C, TRAIT_LIGHT_DRINKER))
 			booze_power *= 2
+		if(isethereal(C)) //alcohol has no effect on Ethereals
```
Comment (coiax) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:44]: Change this to a trait, `TRAIT_BOOZE_IMMUNITY` instead?

Comment (wesoda25) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:44]: Yeah that sounds better

**Comment by coiax** [comment]
> 
> 
> > Floyd mentioned he'd like some interesting ways for Ethereals to regenerate blood. Lmk if you have any ideas for that.
> 
> make it so that drinking alcoholic drinks gives blood with booze power affecting the conversion ratio. So basically, an IV drip of bacchus' blessing will cause you to have 4000% blood.

I think it would make more sense for booze to act as Saline-Glucose Solution, and having Salglu not work on Ethereals (since it's a water based solution).

**Comment by wesoda25** [comment]
I'm probably gonna do that but water is conductive too so it would probably work

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 2]
```diff
@@ -742,25 +742,21 @@
...
-
 /datum/reagent/consumable/liquidelectricity/on_mob_life(mob/living/carbon/M)
-	if(prob(25) && !isethereal(M)) //lmao at the newbs who eat energy bars
-		M.electrocute_act(rand(10,15), "Liquid Electricity in their body", 1, SHOCK_NOGLOVES) //the shock is coming from inside the house
-		playsound(M, "sparks", 50, TRUE, SHORT_RANGE_SOUND_EXTRARANGE)
+	if(isethereal(M))
+		var/obj/item/organ/stomach/ethereal/stomach = M.getorganslot(ORGAN_SLOT_STOMACH)
+		if(istype(stomach))
+			stomach.adjust_charge(REM * 20)
+	else
```
Comment (Qustinnus) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm:755]: change this into else if

**Comment by Learner10** [comment]
Not even 6 days ago, I went on a drunken bender with my Ethereal for New Years, and I had an absolutely fun blast doing it. **This PR gives me the curse of sobriety and I absolutely despise the idea.** On top of that, no more Ethereal chemistry, which means no more emergency stun juice, or home-made thermite.

**Comment by Qustinnus** [comment]
yeah im not gonna lie i dont want them to be immune to alcohol now i think about it more, maybe make them more alcohol immune and make alcoholism a normal thing for them.

**Comment by evilcyborgmrt** [comment]
> I'm probably gonna do that but water is conductive too so it would probably work

Pure water actually doesn't conduct electricity

**Comment by wesoda25** [comment]
> yeah im not gonna lie i dont want them to be immune to alcohol now i think about it more, maybe make them more alcohol immune and make alcoholism a normal thing for them.

Imo an immunity to a system is more interesting than just tolerance - I really like the idea of various drugs affecting different species differently - but I’ll respect your authority on the matter. Maybe later I could just do something where chems which don’t affect humans do ethereals (plus other species). 

If alcohol tolerance is 70% as effective, maybe I’ll go with 30%-40% for ethereals.

**Comment by oklews** [comment]
I hate to see it go, but having tazer blood and the inability to inject more is BAD

**Comment by Krysonism** [comment]
Isopropanol is not very conductive and not very poisonous.

In addition to not making any sense, this removes the most interesting thing about ethereals, their very useful blood.

**Comment by Farquaar** [comment]
This doesn't work from a physiological perspective. Why do lightbulbs have blood to begin with? Alcohol doesn't conduct electricity, so it's not like you can say that it conducts lightbulb electrons like water would. It doesn't really make any sense.

**Comment by wesoda25** [comment]
Bro idk I googled conductive liquids and there it was, figured it would be an interesting change to make them more unique. I’ll go back over this PR with a more critical eye in the coming days and decide where to go with it.

**Comment by wesoda25** [comment]
ok my critical eye says that I am lazy and since this idea wasn't received very well I'll retire it for now

**Pull request closed without merging**